fetch_feature_layer.py

```diff
@@ -30,6 +33,28 @@ def get_column_type(value):
     raise Exception(f'unknown type {type(value)}')
 
 
+def create_point_table_schema(meta_data: MetaData, data: dict, table_name: str, srid: int) -> Table:
+    """
+    """
```

Comment:
Ah yes. The cheater's way of adding code "documentation" so linting won't complain.

Reply:
That method was dumb anyway. I've removed it.